### PR TITLE
added multiple app support in the launch args

### PR DESCRIPTION
**Pull request by mcfatealan**

### CODE REVIEW

**Comment by mcfatealan** [comment]
fix #94 
Now usage:

``` bash
.\dsn.replication.simple_kv.exe config.ini -app_list meta@1,monitor
```

**Comment by imzhenyu** [comment]
good, please update the comments in code/doc as well about -app -app_index

**Comment by mcfatealan** [comment]
Done.

**Comment by mcfatealan** [comment]
Thanks to @qinzuoyan. I forgot to update some places in the codes. Now it's consistent.

**Comment by mcfatealan** [comment]
Wiki in rDSN has been updated too.

**Comment by qinzuoyan** [comment]
Nice, then can merge after checks passed.

File: src/core/core/service_api_c.cpp [hunk 1]
```diff
@@ -924,10 +925,10 @@ DSN_API void dsn_app_loader_wait()
 
 //
 // run the system with arguments
-//   config [-cargs k1=v1;k2=v2] [-app app_name] [-app_index index]
-// e.g., config.ini -app replica -app_index 1 to start the first replica as a new process
-//       config.ini -app replica to start ALL replicas (count specified in config) as a new process
-//       config.ini -app replica -cargs replica-port=34556 to start ALL replicas with given port variable specified in config.ini
+//   config [-cargs k1=v1;k2=v2] [-app_list app_name1@index1,app_name2@index]
```
Comment (imzhenyu) [inline src/core/core/service_api_c.cpp]: use ; as the separator to be consistent with the previous cargs parameters

**Comment by mcfatealan** [comment]
Done. Please review @imzhenyu . Nothing to update in WIKI. #94

**Pull request merged**